Add a hint how to see the dialog box afterwards. (#1620)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/Internationalization/TranslationServer/Crowdin/ExtensionIntegration.rst

   :alt: Result
   :width: 600px

> When you click on Edit button then you will come back into the "Select branches" dialog box from above again.

Reviewer: ```suggestion
Clicking the Edit button will take you back to the `Select Branches` dialog from above.
```
